flutter_vma/flutter_vma.cc

```diff
@@ -16,4 +16,19 @@
 
 #define VMA_IMPLEMENTATION
 
+// Enable this to dump a list of all pending allocations to the log. This comes
+// in handy if you are tracking a leak of a resource after context shutdown.
+#if 0
+#include "flutter/fml/logging.h"
```

Comment:
seems like this import might be causing the compiler to trip-up, unclear why `if 0` isn't stopping it.

Reply:
I think this is a `gn check` instead of a compiler check. I can add no-lint of just add the dependency to flutter_vma.